deepchecks/checks.py

```diff
@@ -62,7 +62,7 @@
     # distribution checks
     'TrainTestFeatureDrift',
     'TrainTestLabelDrift',
-    'WholeDatasetDrift',
+    'MultiVariateDrift',
```

Comment:
I think the name (in all changes) should be `MultivariateDrift` instead of `MultiVariateDrift` (mind the V)

Reply:
Sure, I'll make the necessary changes.